style.css

```diff
@@ -191,6 +191,83 @@ a {
   width: 18px;
 }
 
+.card-container {
+  position: fixed;
+  top: 0;
+  transform: scale(0.1);
+  visibility: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 10px 10px 10px 10px;
+  padding: 10px 10px 10px 10px;
+  background: white;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  color: #172b4d;
+  z-index: 3;
+}
+
+.card-container-title {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.mobiletonic {
+  width: 286px;
+  height: 220px;
+  border-radius: 8px;
+}
+
+.bcontainer {
+  display: flex;
```

Comment:
- [ ] I can see that you have used `display: flex` in so many classes in your CSS file. This is not a good practice as you're repeating yourself. CSS has to follow the [DRY](https://vanseodesign.com/css/dry-principles/) principles. Create a class flex wir display: flex as it's property and pass flex in your HTML.
.flex {
    display: flex;
}
Any element that you want the display to be flex, you just pass the flex class in the html. This reduces repetition and even reduces the length of your code.
- Apply same technique to color as I see the colors repeating.
For example, instead of using color: #3498db; 3 times
   ` #intro1 {color: #3498db; font-size: 10px; font-weight: bold;}`
    `header {color: #3498db; font-size: 20px; background-color: green;}`
   ` #banner {color: #3498db; font-size: 30px; background-image: url(images/static.jpg);}`
you can create a class and add it to all element that need to be blue.
`   .blue {color: #3498db;}`

Reply:
@t-yanick Hello,

I am in a little dark here about what should I do. How could I add .flex { display:flex } in HTML. Is that all optional?

Reply:
Hi @Otman, It is not optional.  Put the flex class in your HTML every element you want to display as flex. Then in CSS create a .flex class with display flex as property